### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lindsey-rix**
We need to point people towards latest version of Stache.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,9 @@
 - `npm install`
 - `bower install`
 - `grunt build`
+
+## New Version of Stache
```
Comment (blackbaud-johnly) [inline README.md:5]: Should this new content be above the existing content? Or maybe replace it altogether? Seems like to me, but maybe it's just me.

Comment (lindsey-rix) [reply README.md:5]: I wasn't sure if I should replace it or not. What do you think?

File: README.md [hunk 2]
```diff
@@ -7,3 +7,9 @@
...
+
+We no longer support development on this version of Stache. Instead, we recommend you check out our latest version of Stache, the Blackbaud Documentation Builder. The currently supported version is a collection of Angular 2 components for SKY UX SPAs. 
+
+https://github.com/blackbaud/stache2
```
Comment (blackbaud-johnly) [inline README.md]: What do you think of removing the URL here and make "our latest version of Stache" a link to the URL? I think that would look cleaner, but it's probably 6 of one, half dozen of the other.

File: README.md (hunk 1 above)
Comment (blackbaud-johnly) [reply README.md:5]: My personal opinion is that we should replace it. I don't feel strongly about it, but replacing that content makes it clear that we don't want people using Stache 1. Leaving the existing content feels like it leaves the door open just a little, which is all some people need.

File: README.md (hunk 2 above)
Comment (lindsey-rix) [reply README.md]: I hadn't really thought either way. Flip a coin. What sounds better to you?

File: README.md (hunk 1 above)
Comment (lindsey-rix) [reply README.md:5]: √ Done, old content removed.

File: README.md (hunk 2 above)
Comment (blackbaud-johnly) [reply README.md]: My preference would be to make "our latest version of Stache" the link. ... But I don't feel strongly about it.

File: README.md [hunk 3]
```diff
@@ -2,8 +2,8 @@
 
 [![npm version](https://badge.fury.io/js/blackbaud-stache.svg)](http://badge.fury.io/js/blackbaud-stache)
 
-## Building Stache
+## New Version of Stache
+
+We no longer support development on this version of Stache. Instead, we recommend you check out our <a href="https://github.com/blackbaud/stache2
+">latest version of Stache</a>, the Blackbaud Documentation Builder. The currently supported version is a collection of Angular 2 components for SKY UX SPAs.
```
Comment (Blackbaud-BobbyEarl) [inline README.md]: I would say, "The currently support version is a collection of Angular components and SDK plugins for the SKY UX ecosystem."

Comment (lindsey-rix) [reply README.md]: I guess I can make that change for you.

**Review by blackbaud-johnly — APPROVED** [review]
LGTM

**Pull request merged**